examples/aggregator/README.md

```diff
@@ -15,24 +15,12 @@ output.
 Build and run the Backend with the following command:
 
 ```bash
+export RUST_LOG=info
```

Comment:
cc @ipetr0v FYI I am making slight changes to these scripts, let me know if you have any comments, I will send you more PRs soon to make this example work with proper labels anyways.